environment.yml

```diff
@@ -7,6 +7,7 @@ dependencies:
   - jupyterlab=4.2.5
   - pygmt=0.13.0
   - python=3.12
-  # Optional dependencies
-  - geopandas=1.0.1
   - jupyter-book=1.0.2
+  # Tutorial 2 - pandas and GeoPandas, Tutorial 4 - Geophysics (Seismology)
+  - pandas=2.2.2
+  - geopandas=1.0.1
```

Comment:
Same as above, maybe better to update the `environment.yml` within one PR for all tutorials to avoid merge conflicts.

Reply:
Since `pandas` is a required dependency of `pygmt`, this won't need to be added here, unless you specifically require `pandas=2.2.2`?

Note that I'll be updating the [`conda-lock.yml`](https://github.com/GenericMappingTools/agu24workshop/blob/main/conda-lock.yml) file to pin all dependencies once everything is stabilized, so that people can reproduce the exact dependency versions in the future (including pandas).

Reply:
I don't think I specifically need this pandas version. Just added the version I am currently having installed in the virtual environment I am using for this pre-workshop.

Reply:
Ok, we can leave the pin on `pandas=2.2.2`, sometimes good to be explicit.